podvm/Makefile.inc

```diff
@@ -102,11 +104,9 @@ $(KATA_AGENT): force $(STATIC_LIBSECCOMP)
 $(STATIC_LIBSECCOMP):
 	$(RUST_FLAGS) $(STATIC_LIBSECCOMP_BUILDER) $(STATIC_LIBSECCOMP_DIR)/kata-libseccomp $(STATIC_LIBSECCOMP_DIR)/kata-gperf
 
-# Skoepo package packages are available in RHEL/CentOS 8 or later and Ubuntu 20.10 or later
-$(SKOPEO_SRC):
+# Skopeo package packages are available in RHEL/CentOS 8 or later and Ubuntu 20.10 or later
+$(SKOPEO_BIN):
```

Comment:
Clould you keep two rules separated, instead of merging them?  I think this is consistent to the rules for other binaries.

```
$(SKOPEO_SRC):
	git clone -b "v$(SKOPEO_VERSION)" "$(SKOPEO_REPO)" "$(SKOPEO_SRC)"

$(SKOPEO_BIN): $(SKOPEO_SRC)
	cd "$(SKOPEO_SRC)" && CC= $(MAKE) bin/skopeo
```

Reply:
Sure, Addressed the change.